Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -187,6 +187,7 @@ typedef NS_ENUM(NSInteger, _WKImmediateActionType) {
 #else
 - (void)_loadAndDecodeImage:(NSURLRequest *)request constrainedToSize:(CGSize)maxSize maximumBytesFromNetwork:(size_t)maximumBytesFromNetwork completionHandler:(void (^)(NSImage *, NSError *))completionHandler WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
 #endif
+- (void)_getInformationFromImageData:(NSData *)imageData completionHandler:(void (^)(NSString *, NSArray<NSValue *> *availableSizes, NSError *error))completionHandler;
```

Comment:
It’s not obvious what the `NSString` is without an argument name. We also need to add `WK_API_AVAILABLE` for this.